infra/kube/gcloud-deploy.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+set -o pipefail
+set -o errexit
+set -o nounset
+# set -o xtrace
+
+
+
+# Set magic variables for current FILE & DIR
+__dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+__kube="${__dir}"
+
+# Store the new image in docker hub
+docker build --quiet -t transloadit/uppy-server:latest -t transloadit/uppy-server:$TRAVIS_COMMIT .;
+docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD";
```

Comment:
Should we make a dedicated user on docker hub for a bot to push this vs being associated with a person?

Reply:
Yeah creating a bot account is good idea we can add it to our secrets also